devops/scripts/benchmarks/history.py

```diff
@@ -161,11 +161,12 @@ def save(self, save_name, results: list[Result], to_file=True):
         os.makedirs(results_dir, exist_ok=True)
 
         # Use formatted timestamp for the filename
-        timestamp = (
-            datetime.now(tz=timezone.utc).strftime("%Y%m%d_%H%M%S")
-            if options.timestamp_override is None
-            else options.timestamp_override
-        )
+        if timestamp is None:
```

Comment:
It seems like the timestamp is unconditionally created in the main() method, so this one is not needed?